06_puppy/hsy3418/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	p1 := Puppy{Breed: "Poodle", Colour: "White", Value: 1280.5}
+	p2 := Puppy{Breed: "Poodle", Colour: "Grey", Value: 1340.5}
+	mapStore := NewMapStore()
+	syncStore := NewSyncStore()
+	_ = mapStore.CreatePuppy(p1)
+	_ = syncStore.CreatePuppy(p2)
+	puppy, _ := mapStore.ReadPuppy(0)
+	puppy2, _ := syncStore.ReadPuppy(0)
+	fmt.Fprintf(out, "Puppy ID %d is %v", puppy.ID, puppy.Value)
```

Comment:
Put a `\n` at the end of this output and the next. This should output a complete line to be user-friendly.